_posts/2018-04-12-vlocchain.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: VlocChain - Front End
+date: 2018-04-12
+categories:
+    - projects
+    - blocckchain
+    - flask
+    - python
+    - jquery
+description: Videos with Blockchain!
+image-sm: https://avatars0.githubusercontent.com/u/36834016?s=400&u=91c5139a84cfe4752a333449e7459883d2518961&v=4
+author: Joe DeGrand
+author-image: https://avatars0.githubusercontent.com/u/33506282?s=400&u=6c92360c9eaa34336fefd3daf438f289295718d5&v=4
+author-bio: First year Software Engineering major at RIT
+author-email: jpd3263@rit.edu
+author-social:
+    github: https://github.com/jdegrand
+    linkedin: https://www.linkedin.com/in/joe-degrand-377906152/
+---
+## What Is VlocChain?
+VlocChain is the concept of sharing videos with others using blockchain technology. Originally thought of at Brick Hack 4, Owen Sullivan, Carter Nesbitt, and myself came up with the idea of making something with blockchain that isn't a cryptocurrency, to show blockchain's potential. Our vision was to create a peer to peer video sharing platform, with no middle man or centralized server involved.
+
+## The Front End
+My part in this unordinary project mostly involved front end based tasks, such as designing and routing webpages, working with databases and makng a user login system, making a custom video web player, working with post and get requests, and working with our main flask app in general.
+
+Before starting this project, he only type of experience that I had was work with python and some slight knowledge on html and css. I now know the basics in flask, mySQL/mariaDB, POST/GET requests, jquery, debugging in nginx/uwsgi, and a bit about jinja. This project has taught me so much already, and I hope that I will continue to learn more as I continue to work on it.
```

Comment:
`he only type` -> `the only type`

Reply:
`some slight knowledge on` -> `slight knowledge about`

Reply:
`html and css` -> `HTML and CSS`, along w/ capitalization of other library and technology names like `NGINX`, `jQuery`, etc.

Reply:
I think you may mean `WSGI` instead of `uwsgi`, could be wrong, though.